### ISSUE TITLE
Northern Barrens: "Mor'shan Caravan Pick-Up"

### ISSUE THREAD

**Issue by Muzantig** [spp-issues/SPP-Legion-Issues#34]
After you start the quest the caravan  NPCs do not appear. This may be tied with the "Crossroads Caravan Delivery" #24 quest being broken.

https://www.wowhead.com/quest=29109/morshan-caravan-pick-up
https://www.wowhead.com/npc=52445/morshan-caravan-pick-up-rocco-credit

![WoWScrnShot_121724_214759](https://github.com/user-attachments/assets/08587c3f-aab4-47aa-bf17-a916359c06ba)

**Comment by skeezerbean** [comment]
Yes, this is phased until the other has been rewarded. And it was a bit messed up, re-scripted this as well and it's now ready.
